x-pack/plugins/apm/public/components/app/TransactionDetails/Transaction/WaterfallContainer/Waterfall/index.tsx

```diff
@@ -38,6 +39,8 @@ export interface IUrlParams {
   transactionType: string;
   transactionName: string;
   errorGroupId: string;
+  flyoutDetailTab: string | null;
+  activeTimelineId: string | null;
```

Comment:
Would this work?

```suggestion
  activeTimelineId?: string;
```

(btw. had to try out the new suggestion feature ;) )

Reply:
Also, you might want to rebase with `apm/dt`. Lots of typescript changes :)

Reply:
Will rebase first thing my time. :)

Reply:
The `?` doesn't work here either because that is an alias for `string | undefined` but not for null. I was using `null` to clear out query param values since you can't remove them easily, since they are merged in with old ones. The `spanId` that exists in 6.4 sets them to null so I went with that. :)